src/pages/Board/index.tsx

```diff
@@ -1,17 +1,33 @@
 import Layout from 'components/Layout';
 import * as S from './style';
-import notifications from 'fixtures/notify.dummy';
 import EyeIcon from 'assets/EyeIcon';
 import MessageIcon from 'assets/MessageIcon';
 import { useNavigate } from 'react-router-dom';
 import theme from 'styles/theme';
 import Input from 'components/Input';
-import { useState } from 'react';
+import { useEffect, useState } from 'react';
+import { useQuery } from 'react-query';
+import { readBoard } from 'apis/api';
+import { IReadBoard } from 'type/IReadBoard';
+import { truncateText } from 'utils/truncateText';
+import { daysAgo } from 'utils/daysAgo';
 
 const Board = () => {
   const [userInput, setUserInput] = useState('');
+  const [projectDetail, setProjectDetail] = useState<IReadBoard[]>();
   const router = useNavigate();
 
+  const { data } = useQuery({
+    queryKey: ['readBoard'],
+    queryFn: readBoard,
+  });
+
+  useEffect(() => {
+    if (data) {
+      setProjectDetail(data.postResponses);
+    }
+  }, [data]);
+
```

Comment:
```suggestion
const GET_BOAR_KEY = "getBoard";

  const getBoardQuery = useQuery({
    queryKey: [GET_BOAR_KEY], // 쿼리키는 상수로 관리
    queryFn: readBoard,
  });

  useEffect(() => {
    if (getBoardQuery.data) {
      setProjectDetail(data.postResponses);
    }
  }, [getBoardQuery.data]);
```

useQuery가 현재는 하나이지만 나중을 고려한다면 여러개로 늘어날수있을것같아요 getBoardQuery와 같이 명시적인 네이밍을 지정해주면 중복될 우려를 대비 할 수 있을 것 같아요

또한 useQuery에서 쿼리키는 매우 중요하기때문에 상수로 관리해줍시다!

Reply:
또 state 네이밍을 보니깐 상세페이지 데이터를 불러오는것같은데 api query에다가 id를 추가하지않아도되는건가요? 만약 상세페이지 쿼리라면 getBoardDetail이 좋을것같아요 상황에 따라 바꿔주세요

Reply:
네 알겠습니다